### PR TITLE
Branch devguide

### PR DESCRIPTION
**Pull request by amoonguss1**
resolves issues #159

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -583,154 +583,141 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-3. User enters the "company {keyword}" command
-4. UI display the list of jobs with companies that contains the keyword.
+**Use case: UC8 - Finding a role by Company name<br>**
+**Actor: User <br>**
+**Preconditions: At least 1 role is added in TechTrack <br>**
+**MSS:**
 
+1. TechTrack is running (UC1).
+2. User enters the "company {keyword}" command.
+3. UI display the list of jobs with companies that contains the keyword.<br>
```
Comment (dsja612) [inline docs/DeveloperGuide.md:709]: Comprehensive coverage of MSS!

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -583,154 +583,141 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-3. UI display the list of jobs with its index.
-4. User enters the “delete {job ID}” to delete the jobs.
-5. UI will respond with the selected jobs being deleted.
+**Use case: UC5 - Delete a role<br>**
+**Actor: User <br>**
+**Preconditions: At least 1 role is added in TechTrack <br>**
+**MSS:**
 
+1. TechTrack is running (UC1).
+2. User enters the “delete {job ID}” to delete the roles.
```
Comment (dsja612) [inline docs/DeveloperGuide.md:667]: we using `role` from now on right?

**Review by dsja612 — CHANGES_REQUESTED** [review]
one nit then lgtm

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (amoonguss1) [reply docs/DeveloperGuide.md]: ok!

**Review by dsja612 — APPROVED** [review]
lgtm

**Pull request merged**